doc/user-guides/multi-cluster.md

```diff
@@ -0,0 +1,289 @@
+  dnsProviderSecretRef:
+    name: aws-credentials
+EOF
+```
+
+Wait for the zone to be ready
+
+```
+k wait managedzone/managedzone --for=condition=ready=true -n ingress-gateway
+```
+
+
+### Add a TLS Issuer
+
+To secure our gateways we want to define a TLS issuer for TLS certificates. We will use letsencrypt, but you can use any supported by cert-manager.
+
+
+```
+export EMAIL=myemail@email.com
+```
+```
+kubectl apply -f - <<EOF
+apiVersion: cert-manager.io/v1
+kind: ClusterIssuer
+metadata:
+  name: lets-encrypt
+spec:
+  acme:
+    email: ${EMAIL} 
+    privateKeySecretRef:
+      name: le-secret
+    server: https://acme-staging-v02.api.letsencrypt.org/directory
+    solvers:
+      - dns01:
+          route53:
+            hostedZoneID: ${zid}
+            region: us-east-1
+            accessKeyIDSecretRef:
+              key: AWS_ACCESS_KEY_ID
+              name: aws-credentials
+            secretAccessKeySecretRef:
+              key: AWS_SECRET_ACCESS_KEY
+              name: aws-credentials
+EOF
+```
+
+### Setup a Gateway
+
+In order for Kuadrant to balance traffic using DNS across two or more clusters. We need to define a gateway with a shared host. We will define this as a HTTPS listener with a wildcard DNS entry based on your root domain. As mentioned, these resources need to be applied to both clusters.
+
+```
+kubectl apply -f - <<EOF
+apiVersion: gateway.networking.k8s.io/v1
+kind: Gateway
+metadata:
+  name: external
+  namespace: ingress-gateway
+spec:
+    gatewayClassName: istio
+    listeners:
+    - allowedRoutes:
+        namespaces:
+          from: All
+      hostname: "*.${rootDomain}"
+      name: api
+      port: 443
+      protocol: HTTPS
+      tls:
+        certificateRefs:
+        - group: ""
+          kind: Secret
+          name: api-external-tls
+        mode: Terminate
+EOF
+```        
+
+### Secure and Protect the Gateway with Rate Limiting, Auth and TLS
+
+While our gateway is deployed, it is not yet exposed via an address. This is because we have not yet setup the TLS cert and so the Gateway is not in a ready state. Before we do that lets set up an `AuthPolicy` so that when our gateway is exposed any endpoint will default to a `DENY ALL` 403 response. We will also setup a `ratelimitpolicy` and  `TLSPolicy` to setup our certificates.
+
+```
+kubectl apply -f - <<EOF
+apiVersion: kuadrant.io/v1beta2
+kind: AuthPolicy
+metadata:
+  name: external
+  namespace: ingress-gateway
+spec:
+  targetRef:
+    group: gateway.networking.k8s.io
+    kind: Gateway
+    name: external
+  defaults:
+    rules:
+      authorization:
+        "deny":
+          opa:
+            rego: "allow = false"
+EOF
+```
+
+RateLimitPolicy
+
+```
+kubectl apply -f  - <<EOF
+apiVersion: kuadrant.io/v1beta2
+kind: RateLimitPolicy
+metadata:
+  name: low-limit
+  namespace: ingress-gateway
+spec:
+  targetRef:
+    group: gateway.networking.k8s.io
+    kind: Gateway
+    name: external
+  overrides:
```

Comment:
Maybe add some explanation here that, by creating an *override* RateLimitPolicy, the limits here declared will be enforced on all routes despite more specific policies occasionally attached to the routes later on (which is actually the case.)